Dockerfile

```diff
@@ -42,8 +42,8 @@ RUN \
   git config --global user.name "Builder" && \
   git config --global user.email "builder@localhost"
 
-ARG BRVER="2021.02.3"
-ARG KVER="5.4.125"
+ARG BRVER="2022.05.2"
+ARG KVER="5.10.129"
```

Comment:
Is this version of the kernel tightly coupled with the buildroot version?

Reply:
It's the kernel buildroot takes the kernel headers from, used e.g. for glibc and musl. Each buildroot release has some predefined versions and hashes for them that were current at the time. We need to take the earliest kernel still featured in any Bottlerocket release, but otherwise don't need to worry about updating since the kernel ABI won't/is not supposed to break. (One thread to pull on: https://git.busybox.net/buildroot/tree/package/linux-headers/Config.in.host?h=2022.02.5#n382)